### PR TITLE
[release] 20211216

### PR DESCRIPTION
**Pull request by jiqiang90**

### CODE REVIEW

**Comment by wqsz7xn** [comment]
This is going to have to be done again to include https://github.com/subquery/subql/commit/f67ed0aae7e054617cccf3962c9b2fda99aa832b

File: packages/cli/CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,10 @@ All logs must start with the format: [x.y.z] - yyyy-mm-dd
 
 ## [Unreleased]
 
+## [0.16.2] - 2021-12-16
+### Fixed
+- Flyby fix, adjust jsonrpc timeout (#640)
```
Comment (ianhe8x) [inline packages/cli/CHANGELOG.md:13]: no need to mention it, the timeout change is on tests.

File: packages/node/CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.26.0] - 2021-12-16
+### Added
+- Support dictionary for custom datasource (#686)
+- Automatic adjust fetched block batch size based on memory usage (#668)
```
Comment (ianhe8x) [inline packages/node/CHANGELOG.md:12]:
```suggestion
- Automatic adjust fetched block batch size based on memory usage, enable by passing --scale-batch-size (#668)
```

File: packages/query/CHANGELOG.md [hunk 3]
```diff
@@ -8,6 +8,16 @@ All logs must start with the format: [x.y.z] - yyyy-mm-dd
 
 ## [Unreleased]
 
+## [0.9.0] - 2021-12-16
+### Changed
+- Limit query record size to 100, by using `unsafe` remove this limit. (#644)
```
Comment (ianhe8x) [inline packages/query/CHANGELOG.md:13]:
```suggestion
- Limit query record size to 100, though by using `--unsafe` can remove this limit. (#644)
```

File: packages/validator/CHANGELOG.md [hunk 4]
```diff
@@ -9,6 +9,10 @@ All logs must start with the format: [x.y.z] - yyyy-mm-dd
 
 ## [Unreleased]
 
+## [0.4.1] - 2021-12-16
+### Fixed 
+- Flyby fix, update test (#686)
```
Comment (ianhe8x) [inline packages/validator/CHANGELOG.md:14]: doesn't look like need a release

File: packages/cli/CHANGELOG.md (hunk 1 above)
Comment (ianhe8x) [reply packages/cli/CHANGELOG.md:13]: doesn't look like need a release.

**Review by ianhe8x — APPROVED** [review]

**Pull request merged**